Sources/Testing/Testing.docc/LimitingExecutionTime.md

```diff
@@ -0,0 +1,64 @@
+# Limiting the execution time of a test
+
+<!--
+This source file is part of the Swift.org open source project
+
+Copyright (c) 2023 Apple Inc. and the Swift project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+Set limits on how long a test can run for until it fails.
+
+## Overview
+
+Some tests may naturally run slowly: they may require significant system
+resources to complete, may rely on downloaded data from a server, or may
+otherwise be dependent on external factors.
+
+If a test may hang indefinitely or may consume too many system resources to
+complete effectively, consider setting a time limit for it so that it will
+be marked as failing if it runs for an excessive amount of time. Use the
+``Trait/timeLimit(_:)`` trait as an upper bound:
+
+```swift
+@Test(.timeLimit(.seconds(60 * 60)) func serve100CustomersInOneHour() async {
```

Comment:
```suggestion
@Test(.timeLimit(.seconds(60 * 60))
 func serve100CustomersInOneHour() async {
```

Reply:
Whitespace is wonky, will revise manually.